doc/classes/Object.xml

```diff
@@ -601,43 +731,45 @@
-				[b]Note:[/b] Negative and floating-point values usually represent physical entities for which singular and plural don't clearly apply. In such cases, use [method tr].
-				See [url=$DOCS_URL/tutorials/i18n/localization_using_gettext.html]Localization using gettext[/url] for examples of the usage of this method.
+				Translates a [param message] or [param plural_message], using the translation catalogs configured in the Project Settings. Further [param context] can be specified to help with the translation.
+				If [method can_translate_messages] is [code]false[/code], or no translation is available, this method returns [param message] or [param plural_message], without changes. See [method set_message_translation].
+				The [param n] is the number, or amount, of the message's subject. It is used by the translation system to fetch the correct plural form for the current language.
+				For detailed examples, see [url=$DOCS_URL/tutorials/i18n/localization_using_gettext.html]Localization using gettext[/url].
+				[b]Note:[/b] Negative and [float] numbers may not properly apply to some countable subjects. It's recommended handling these cases with [method tr].
 			</description>
 		</method>
 	</methods>
 	<signals>
 		<signal name="property_list_changed">
 			<description>
+				Emitted when [method notify_property_list_changed] is called.
 			</description>
 		</signal>
 		<signal name="script_changed">
 			<description>
-				Emitted whenever the object's script is changed.
+				Emitted when the object's script is changed.
+				[b]Note:[/b] When this signal is emitted, the new script is not initialized yet. If you need to access the new script, defer connections to this signal with [constant CONNECT_DEFERRED].
 			</description>
 		</signal>
 	</signals>
 	<constants>
 		<constant name="NOTIFICATION_POSTINITIALIZE" value="0">
-			Called right when the object is initialized. Not available in script.
+			Notification received when the object is initialized, before its script's [method _init] is called. Used internally.
 		</constant>
 		<constant name="NOTIFICATION_PREDELETE" value="1">
-			Called before the object is about to be deleted.
+			Notification received when the object is about to be deleted. Can act as the deconstructor of some programming languages.
 		</constant>
 		<constant name="CONNECT_DEFERRED" value="1" enum="ConnectFlags">
-			Connects a signal in deferred mode. This way, signal emissions are stored in a queue, then set on idle time.
+			Deferred connections trigger their [Callable]s on idle time, rather than instantly.
 		</constant>
 		<constant name="CONNECT_PERSIST" value="2" enum="ConnectFlags">
-			Persisting connections are saved when the object is serialized to file.
+			Persisting connections are stored when the object is serialised (such as when using [method PackedScene.pack]). In the editor, connections created through the Node dock are always persisting.
```

Comment:
```suggestion
			Persisting connections are stored when the object is serialized (such as when using [method PackedScene.pack]). In the editor, connections created through the Node dock are always persisting.
```

Since all other places spell this way.